internal/handlers/sqlite/msg_create.go

```diff
@@ -79,6 +70,44 @@ func (h *Handler) MsgCreate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		return nil, err
 	}
 
+	params := backends.CreateCollectionParams{
+		Name: collectionName,
+	}
+
+	capped, err := common.GetOptionalParam[bool](document, "capped", false)
+	if err != nil {
+		return nil, err
+	}
+
+	if capped {
+		// size param is required
```

Comment:
So why is that a comment and not `common.GetRequiredParam`?

Reply:
`GetRequiredParam` supports only one exact data type, but size can be set as int32, int64, float etc.